### PR TITLE
[Swift] Container operations

### PR DESCRIPTION
**Pull request by outcatcher**
Add new `SwiftV1` service

Implement container and account operations

Depends-On: https://github.com/opentelekomcloud/oms/pull/57

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

File: src/oms/services/swift/v1/types.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export interface Container {
+    readonly count: number
+    readonly bytes: number
+    readonly name: string
+    readonly last_modified?: string
+}
+
+export interface Account {
+    readonly domainID: string
```
Comment (lego963) [inline src/oms/services/swift/v1/types.ts:11]: did you miss projectID?

File: src/oms/services/swift/v1/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    async listContainers(): Promise<Containers> {
+        return await listContainers(this.client)
+    }
+
+    async deleteContainer(name: string): Promise<void> {
+        return await deleteContainer(this.client, name)
+    }
+
```
[the remark this answers is not in the document]
Comment (outcatcher) [reply src/oms/services/swift/v1/index.ts:51]: yes

File: src/oms/services/swift/v1/types.ts (hunk 1 above)
Comment (outcatcher) [reply src/oms/services/swift/v1/types.ts:11]: It is not returned:

File: tests/functional/services/swift.test.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    const containers = await srv.listContainers()
+    expect(containers).toBeDefined()
+    expect(containers).toHaveProperty('length')
+})
+
+test('Containers: workflow', async () => {
+    const srv = client.getService(SwiftV1)
+    const name = randomString(10)
+    await srv.createContainer(name, undefined, { 'container-meta': 'yes' })
+    let containers = await srv.listContainers()
```
Comment (lego963) [inline tests/functional/services/swift.test.ts:59]: what about `getContainer()` against `listContainers()`

**Review by lego963 — APPROVED** [review]

File: tests/functional/services/swift.test.ts (hunk 3 above)
Comment (outcatcher) [reply tests/functional/services/swift.test.ts:59]: Didn't want to put different endpoints into one test

**Pull request merged**